core/native/src/main/scala/bobcats/evp.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bobcats
+
+import scala.scalanative.unsafe._
+import scala.annotation.nowarn
+
+@extern
+@link("crypto")
+@nowarn
```

Comment:
I don't understand why I need a `@nowarn` here. I note that you don't have it in `fs2-io`, but I can't find the relevant setting and the commit which removed the annotation doesn't have anything at all!

See [here](https://github.com/typelevel/fs2/pull/3014).